### PR TITLE
Implementing some comments

### PR DESCRIPTION
**Pull request by lfoffano**
It implements some comments by Thomas Vuillaume  concerning docstrings, variable names, and comments left in the code

### CODE REVIEW

File: lstchain/analysis/post_dl2.py [hunk 1]
```diff
@@ -78,8 +78,8 @@ def analyze_wobble(config):
         n_off += np.sum(named_datasets[-1][1] < theta2_cut)
 
     stat = WStatCountsStatistic(n_on, n_off, 1./(n_points - 1))
-    lima_significance = stat.significance.item()
-    lima_excess = stat.excess
+    lima_significance = stat.sqrt_ts.item()
```
Comment (mexanick) [inline lstchain/analysis/post_dl2.py:89]: What motivates this? Can you please explain? You change here the definition of excess significance from Li&Ma to sqrt(S/B)...

File: lstchain/io/io.py [hunk 2]
```diff
@@ -1071,7 +1071,7 @@ def read_dl2_to_pyirf(filename):
     return events, pyirf_simu_info
 
 
-def read_dl2_sub_run(t_filename, columns_to_read=None):
+def read_dl2_params(t_filename, columns_to_read=None):
```
Comment (mexanick) [inline lstchain/io/io.py]: This should also be updated at the function list on top of the file, l46

File: lstchain/analysis/post_dl2.py [hunk 3]
```diff
@@ -69,23 +69,23 @@ def analyze_wobble(config):
...
     stat = WStatCountsStatistic(n_on, n_off, 1./(n_points - 1))
     lima_significance = stat.sqrt_ts.item()
     lima_excess = stat.n_sig
-    LOGGER.info('Observation time %s', observation_time)
-    LOGGER.info('Number of "ON" events %s', n_on)
-    LOGGER.info('Number of "OFF" events %s', n_off)
-    LOGGER.info('ON/OFF observation time ratio %s', 1./(n_points - 1))
-    LOGGER.info('Excess is %s', lima_excess)
-    LOGGER.info('Li&Ma significance %s', lima_significance)
+    LOGGER.info('Observation time: {:.1f} s'.format(observation_time))
```
Comment (mexanick) [inline lstchain/analysis/post_dl2.py]: Please remove the formatting change. Pythonic logger uses lazy formatting if implemented as I did, but this won't be the case with your formatting and the expressions will be always evaluated independently of log level

**Review by mexanick — CHANGES_REQUESTED** [review]

File: lstchain/analysis/post_dl2.py (hunk 1 above)
Comment (lfoffano) [reply lstchain/analysis/post_dl2.py:89]: That problem comes from the version of gammapy. The new version of gammapy requested for this new version of lstchain does not have those attributes. I updated them
https://docs.gammapy.org/dev/api/gammapy.stats.WStatCountsStatistic.html

File: lstchain/io/io.py (hunk 2 above)
Comment (lfoffano) [reply lstchain/io/io.py]: true, thanks

File: lstchain/analysis/post_dl2.py (hunk 3 above)
Comment (lfoffano) [reply lstchain/analysis/post_dl2.py]: OK done

File: lstchain/analysis/post_dl2.py (hunk 1 above)
Comment (mexanick) [reply lstchain/analysis/post_dl2.py:89]: Can you link the relevant discussion please? Also a comment in the code with a reference to implementation is highly appreciated

Comment (lfoffano) [reply lstchain/analysis/post_dl2.py:89]: Which discussion? The code, implemented in the new version of lstchain, does not work. That solves the problem, it is only an attribute problem. I do not understand what else to do, sorry. What should we do in this case?

Comment (mexanick) [reply lstchain/analysis/post_dl2.py:89]: ah, so this is just an API (method signature) change? If so, please just add a note about it in the code with a link to gammapy doc as you pointed out above.

Comment (lfoffano) [reply lstchain/analysis/post_dl2.py:89]: OK done. Thanks

**Pull request merged**